index.bs

```diff
@@ -1798,6 +1822,66 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Client Supported Capabilities (dictionary <dfn dictionary>SupportedCapabilities</dfn>) ### {#dictionary-supportedcapabilities}
+
+<xmp class="idl">
+    dictionary SupportedCapabilities {
+        boolean areU2FDevicesSupported;
+        boolean areCTAPDevicesSupported;
```

Comment:
Can you explain the user flow for how you plan to use `areU2FDevicesSupported` and `areCTAPDevicesSupported`?

Reply:
CTAP devices are not supported on every browser.  RP will look into this field to decide what they want to support and what they want to show on their page.  User Agent detection is too fragile to make these decisions and it requires constant monitoring of what browser/platform combination works.

Reply:
But why do RPs care about CTAP2 vs CTAP1 (U2F)?  They care about *capabilities* and *features* -- not transports.  Both of which are covered below.

Reply:
Agreed that we do not currently have a use case for such granularity.  I have removed these particular capabilities.